ci-operator/step-registry/assisted/baremetal/conf/user-data/assisted-baremetal-conf-user-data-commands.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ baremetalds assisted baremetal conf user-data command ************"
+
+cat "${SHARED_DIR}/user-data" << EOF
```

Comment:
should we also save it as an artifact?

Reply:
I expect it to be in the sos report, I need to check that